### ISSUE TITLE
feat: searchbar is performing unwanted autocorrect

### ISSUE THREAD

**Issue by pmuston** [driftyco/ionic#5749]
**Type**: <span ionic-type>feat</span>

**Ionic Version**: <span ionic-version>2.x</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>The SearchBar component is performing unwanted autocorrection. The search terms are often fragments of text - like a couple of letters. Therefore should NOT be autocorrected etc. I need to be able to set options like can be done for input tag- 
`<input autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" />`
</span>

<span is-issue-template></span>

**Comment by adamdbradley** [comment]
Good call! I've added those attributes, thanks

**Comment by jgfet** [comment]
I'm currently experiencing similar unwanted behaviour like this for the ion-searchbar.
I get something like sentence case, so first character is uppercase, then it flips to lower.
I was initially thinking why is it when I sign into Skype on an iphone does the shift key started greyed out, yet my Ionic app doesn't. Then came across a post on [Stack overflow](https://stackoverflow.com/questions/5171764/how-do-you-turn-off-auto-capitalisation-in-html-form-fields-in-ios/5171812#comment75599034_5171812), that others seemed to be able to have success with. Tried adding the attributes to the ion-searchbar. It didn't attach the attributes to the nested input. So I tried dom manipulation and couldn't get autocapitalize="none" to work. (also tried autocorrect="off).
So I wonder if this bug has somehow gotten re-introduced.
ionic info

global packages:

    @ionic/cli-utils : 1.3.0
    Cordova CLI      : 7.0.1
    Ionic CLI        : 3.3.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.3.0
    @ionic/cli-plugin-ionic-angular : 1.3.0
    Cordova Platforms               : android 6.2.3 ios 4.4.0
    Ionic Framework                 : ionic-angular 3.2.1

System:

    Node       : v7.10.0
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.2 Build version 8E2002
    ios-deploy : 1.9.1
    ios-sim    : 5.0.13

/opt/IonicProjects/aewsmob>
